examples/stargate-jwt-auth/start_jwt.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+export CASSTAG=3.11.8
+export SGTAG=v1.0.7
+export KCTAG=11.0.2
+
+# Make sure cass-1, the seed node, is up before bringing up other nodes and stargate
+docker-compose up -d cass-1
+# Wait until the seed node is up before bringing up more nodes
+(docker-compose logs -f cass-1 &) | grep -q "Created default superuser role"
+
+# Bring up the 2nd C* node
+docker-compose up -d cass-2
+(docker-compose logs -f cass-1 &) | grep -q "is now part of the cluster"
+# Bring up the 3rd C* node
+docker-compose up -d cass-3
+(docker-compose logs -f cass-1 &) | grep -q "is now part of the cluster"
+
+# this is where a cql script must be run to insert data and grant permissions
+docker run --rm -it -v /Users/lorina.poland/CLONES/stargate/docker-images/examples/stargate-jwt-auth/scripts:/scripts  \
+-v /Users/lorina.poland/CLONES/stargate/docker-images/examples/stargate-jwt-auth/cqlshrc:/.cassandra/cqlshrc  \
+--env CQLSH_HOST=host.docker.internal --env CQLSH_PORT=9045  nuvo/docker-cqlsh
```

Comment:
Path's need updated.

Reply:
This is outdated. I added a variable to use for the path to the git clone directory in the script. The script unfortunately requires an absolute path, and cannot use a relative path.